python/ray/tests/test_cli.py

```diff
@@ -635,6 +635,155 @@ def test_ray_start_block_and_stop(configure_lang, monkeypatch, tmp_path, cleanup
         worker_proc.kill()
 
 
+@contextmanager
```

Comment:
This file is already too big. Suggest to create a separate `test_symmetric_run.py`